spec/index.html

```diff
@@ -2263,6 +2201,69 @@ <h2>Acknowledgments</h2>
 
 </section>
 
+<section id="section-Changes" class="informative appendix">
+  <h2>Substantive Changes</h2>
+
+<section id="ChangeLog-11" class="informative appendix" >
+  <h2>Substantive changes between RDF 1.0 and RDF 1.1 </h2>
+
+<ul>
+
+ <li>
+    The RDF 1.0 semantics defined simple interpretations relative to a vocabulary.
+  </li>
+
+  <li>
+    In the RDF 1.0 semantics, IL was a total, rather than partial, mapping.
+  </li>
+  
+  <li> The RDF 1.0 specification divided literals into 'plain' literals
+      with no type and optional language tags, and typed literals.
+      Usage has shown that it is important that every literal have a type.
+      RDF 1.1 replaced plain literals without language tags by literals typed with
+      the XML Schema <code>string</code> datatype,
+      and introduced the special type
+      <a data-cite="RDF12-CONCEPTS#dfn-language-tagged-string"><code>rdf:langString</code></a>
+      for language-tagged strings.
+      The full semantics for typed literals is given in section [[[#datatypes]]].
+  </li>
+
+  <li>In the RDF 1.0 specification,
+    datatype D-entailment was defined as a <a>semantic extension</a> of RDFS-entailment.
+    In RDF 1.1 it was defined as a direct extension to basic RDF.
```

Comment:
```suggestion
    In RDF 1.1, it was defined as a direct extension to basic RDF.
```

Reply:
I do not view this as a desirable change.

Reply:
What's your argument against adding this comma? Perhaps my reason for adding it will be made clearer if you look at the preceding sentence, which starts `In the RDF 1.0 specification,`.

Reply:
I removed the comma from the previous sentence.

Reply:
That's not the correct choice. The "In RDF 1.1" and "In the RDF 1.0 specification" are each an independent clause, and should be followed by a comma when they start the sentence. They *could* be moved to the end where they need not be preceded by a comma (e.g., `Datatype D-entailment was defined as a <a>semantic extension</a> of RDFS-entailment in the RDF 1.0 specification. It was defined as a direct extension to basic RDF in RDF 1.1.`), but this would make the meanings of the sentences less clear.